core/lib/libtomcrypt/sha512_accel.c

```diff
@@ -0,0 +1,183 @@
+/* LibTomCrypt, modular cryptographic library -- Tom St Denis */
+/* SPDX-License-Identifier: Unlicense */
```

Comment:
Shouldn't this be:
```C
// SPDX-License-Identifier: Unlicense AND BSD-2-Clause
/* LibTomCrypt, modular cryptographic library -- Tom St Denis */
/* Copyright (c) 2023 Linaro Limited */
...
```

...since the file is originally from LTC then modified by us?

Reply:
I'll fix.